src/cirq_iqm/adonis.py

```diff
@@ -16,20 +16,23 @@
 """
 from __future__ import annotations
 
+from math import pi as PI
 from typing import Optional
 
 import cirq
 from cirq import ops
 
 import cirq_iqm.iqm_device as idev
-import cirq_iqm.iqm_gates as ig
+
+
+PI_2 = PI / 2
 
 # common gates used in gate decompositions
 CZ = ops.CZPowGate()
-Lx = ops.XPowGate(exponent=0.5, global_shift=-0.5)
-Lxi = ops.XPowGate(exponent=-0.5, global_shift=-0.5)
-Ly = ops.YPowGate(exponent=0.5, global_shift=-0.5)
-Lyi = ops.YPowGate(exponent=-0.5, global_shift=-0.5)
+Lx = ops.rx(PI_2)
+Lxi = ops.rx(-PI_2)
+Ly = ops.ry(PI_2)
+Lyi = ops.ry(-PI_2)
 
```

Comment:
Are these new definitions compatible with the current `iqm_operation_mapping.py` (which is not in this PR but is already in master)? Or will a circuit consisting of these gates evne be assumed to be executable without some further processing?

Reply:
rx and ry are convenience functions that return XPowGate and YPowGate instances, but use a more common parameterization.